### ISSUE TITLE
Fully remove boost dependencies

### ISSUE THREAD

**Issue by FabienDanieau** [libnifalcon/libnifalcon#44]
There is still a dependency to boost in falcon/gmtl/Misc/MatrixConvert.h

**Comment by qdot** [comment]
+1 :)

**Comment by FabienDanieau** [comment]
It seems that only vectors and matrices are used from GMTL. I think we can find / do something lighter than having the whole library included.

**Comment by qdot** [comment]
Yeah, I honestly can't remember why I included it other than "it was there and did what I needed at the time".

**Comment by FabienDanieau** [comment]
Well basically  falcon/gmtl/Misc/MatrixConvert.h is not used anywhere so there is no really a dependency to boost. I guess we can leave it as it is
